README.md

```diff
@@ -1,4 +1,58 @@
 # aze-cli
 
-### Commands
-- aze-cli init --player <PLAYER_IDS> --small_blind <SMALL_BLIND_AMOUNT>
\ No newline at end of file
+## Commands
+- ### aze-cli init
+  Initializes a new game.
+  
+  **Arguments:**
+    - `game_type`: Only `Holdem` is supported for now.
+    - `player`: Array containing account ids of the players for current game.
+    - `small_blind`: Small blind amount for the current game.
+    - `buy_in`: Buy in amount for the current game.
+    - `config`: An optional `Config.toml` file containing all the above data.
+
+  **Example usage:**
+    - Without a `Config.toml`
+      ```sh
+      aze-cli init -g Holem -p id1 id2 id3 id4 -s 5 -b 1000
+    - With a `Config.toml`
+      ```sh
+      aze-cli init -c ./Config.toml
+
+- ### aze-cli register
+  Creates a player account.
+
+  **Arguments:**
+    - `identifier`: A string identifier for mapping to its corresponding `AccountId`
+
+  **Example usage:**
+  ```sh
```

Comment:
I think you forgot to add the `connect` command. I think @Abhay-2811 can help here

Reply:
Done